Phase-5/SearchEngine/Interfaces/IManager.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace SearchEngine.Interfaces
+{
+    public interface IManager
+    {
+        public void Run();
+        void MakeSearchEngine();
+        void MakeInvertedIndex(string path);
```

Comment:
these methods can be a private method of 'Manager' and be removed from the interface.
I think users shouldn't call them